### DISCUSSION TITLE
Can build some example projects, but not others.

### DISCUSSION THREAD

**Discussion by provoke** [skiptools/skip#90] [Q&A] [answered]
First of all, this is such a cool tool and I'm excited to use it on real world projects! Will be a customer for sure.

I've tried an Intel Mac and an M2 Mac and have the same experience. I'm able to build the **Hello Skipper** and **SwiftUI Showcase**, but unable to build the others during the Gradle stage:

GRADLE> * What went wrong:

GRADLE> Script compilation errors:

GRADLE> 

GRADLE>   Line 28:     alias(libs.plugins.kotlin.android)

GRADLE>                      ^ Unresolved reference: libs

GRADLE> 

GRADLE>   Line 29:     alias(libs.plugins.android.library)

GRADLE>                      ^ Unresolved reference: libs

GRADLE> 

GRADLE> 2 errors

It looks like the other projects are using plugin aliases, but I'm not exactly sure how to fix.

Please let me know if there's any additional info I can provide.

**Comment by aabewhite** [comment, marked as answer]
Sorry for the trouble! We're in the process of updating our Android toolchain, and you may have gotten caught in the middle. Please try:

> brew update
> brew upgrade skip

Then get the latest source of whatever sample you want to build and try again. Please let us know if things still don't work!

**Reply by provoke** [reply]
Thanks! All good now.
